theia-rust-docker/README.md

```diff
@@ -0,0 +1,44 @@
+# Theia with Rust support
+
+## Building the image
+
+<pre>
+cd <this repo>/theia-rust-docker
```

Comment:
I think you need to escape the `<` and `>` characters in MarkDown. ATM the rendering looks like:
![image](https://user-images.githubusercontent.com/25749063/61530085-166d8700-a9f1-11e9-97e3-3310115b214e.png)

Reply:
Updated